apps/dashboard/src/data/action-types.js

```diff
@@ -5,3 +5,5 @@ export const POLL_UPDATE = 'POLL_UPDATE';
 export const PROJECT_SAVE = 'PROJECT_SAVE';
 export const PROJECT_UPDATE = 'PROJECT_UPDATE';
 export const ROUTE_UPDATE = 'ROUTE_UPDATE';
+export const FETCH_ONGOING = 'FETCH_ONGOING';
```

Comment:
This is pretty bad naming. Like, even in the beta state we'll already have scenarios with multiple requests happening at the same time. And not all of them are made equal hence these action names should be specific to the resource they affect.

Also, I never added the comment here since we've been doing it on the dashboard but it helps to follow a similar naming scheme for all of these, that being: `RESOURCE_NAME_ACTION`. Hence for example `ROUTE_UPDATE` being a thing instead of just `REDIRECT`.
And alphabetical order please.

`PROJECT_SAVE` was intended for this very purpose in this scenario - and as such made separate from `PROJECT_UPDATE`. Likewise you'd use something like `PROJECT_FETCH` if you're just retrieving an existing project for editing or otherwise.

Makes it much easier to browse and you can bet on this file growing a lot in the future.

Reply:
Yeah, I hear you. I was thinking on a single on/off thing in case (eventually) we found ourselves doing independent and separate requests (title comes to mind), so having a single action that just switches the *fetching state* (hence, I created the action creator on `ui`) made sense. But I agree with the usual use of this and will change it.

Reply:
Yeah... if we have separate request, then it might actually be best to manage that part of the state separately as well. Just to avoid dealing in any kind of race conditions at all.